Say Their Names/Source/View/Home/Person/PersonCell.swift

```diff
@@ -8,11 +8,28 @@
 
 import UIKit
 
-class PersonCell: UICollectionViewCell {
+final class PersonCell: UICollectionViewCell {
 
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var imgView: UIImageView!
+    @IBOutlet weak var dobLabel: UILabel!
+    @IBOutlet weak var bookmarkButton: UIButton!
+    
+    var data: Person? {
+        didSet {
+            //guard let data  = data else {return}
+        }
+    }
+    
+    // MARK:- Lifecycle
+    
     override func awakeFromNib() {
         super.awakeFromNib()
-        // Initialization code
     }
 
+    // MARK:- Handlers
+        
+    @IBAction func bookmarkButtonDidTouchUpInsde(_ sender: Any) {
```

Comment:
You don't need to specify everything you are doing in the name of the method. 
First, this is a function, we know that functions are actions.
Second, the sender itself has the type of element used here (which must be UIButton instead of Any).
Third, if you don't want to make it available outside of the class, it must be private. Most of the time, this kind of method will be private.

```suggestion
    @IBAction private func bookmark(_ sender: UIButton) {
```